docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<p>Since <code>**kwargs</code> are the arguments passed to <code>timeit</code>, it would help if the name of this function was something like <code>timeit_pandas_cudf</code>. Perhaps we should also include a docstring?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Done 👍